spec/wings/valkyrie/query_service_spec.rb

```diff
@@ -304,6 +310,69 @@ class Image < ActiveFedora::Base
     end
   end
 
+  describe ".find_inverse_references_by" do
+    context "when the resource is saved" do
+      context "when the property is unordered" do
+        it "returns everything which references the given resource by the given property" do
+          parent = persister.save(resource: resource_class.new(title: ['parent']))
+          parent2 = persister.save(resource: resource_class.new(title: ['parent2']))
+          child = persister.save(resource: resource_class.new(title: ['child'], a_member_of: [parent.id]))
+          child2 = persister.save(resource: resource_class.new(title: ['child2'], a_member_of: [parent.id, parent2.id, parent.id]))
+          persister.save(resource: resource_class.new(title: ['resource']))
+          persister.save(resource: image_resource_class.new(title: ['resource 2']))
+
+          expect(query_service.find_inverse_references_by(resource: parent, property: :a_member_of).map(&:id).to_a).to contain_exactly child.id, child2.id
+        end
+
+        it "returns an empty array if there are none" do
+          parent = persister.save(resource: resource_class.new(title: ['parent']))
+
+          expect(query_service.find_inverse_references_by(resource: parent, property: :a_member_of).to_a).to eq []
+        end
+      end
+
+      # not yet supported by the wings persister
+      context "when the property is ordered" do
+        xit "returns everything which references the given resource by the given property" do
```

Comment:
@hackmastera Did you intend for this to have `xit`

Reply:
@jeremyf yes it's still pending because ordered properties aren't passing persister specs. It may be that this will never be supported for the hyrax / wings adapter, but the initial phase is to get as many of valkyrie's shared specs passing as we can, and keeping this in as pending will help us track how we did with that.